src/SearchParameters/index.js

```diff
@@ -1139,12 +1140,15 @@ SearchParameters.prototype = {
    * contains any refinement value.
    * @method
    * @param {string} attribute attribute for which the refinement is applied
-   * @param {string} operator operator of the refinement
+   * @param {string} [operator] operator of the refinement
    * @param {string} [value] value of the refinement
    * @return {boolean} true if it is refined
    */
   isNumericRefined: function isNumericRefined(attribute, operator, value) {
     if (isUndefined(value)) {
+      return !!this.numericRefinements[attribute];
+    }
+    if (isUndefined(value)) {
```

Comment:
Hmm, this check will never pass, because you already checked for that and returned above